internal/logger/target/http/http.go

```diff
@@ -343,27 +340,25 @@ func (h *Target) startQueueProcessor(ctx context.Context, mainWorker bool) {
 	globalBuffer := logChBuffers[name]
 	logChLock.Unlock()
 
-	newTicker := time.NewTicker(time.Second)
-	isTick := false
 	var count int
-
 	for {
-		isTick = false
-		select {
-		case _ = <-newTicker.C:
-			isTick = true
-		case entry, _ = <-globalBuffer:
-		case entry, ok = <-h.logCh:
-			if !ok {
+		var (
+			ok    bool
+			entry any
+		)
+
+		if count < h.batchSize {
+			select {
```

Comment:
it looks like a ticker is still needed here, because there is a possiblity that one event is kept in the queue and not sent for a long time when there is no new entries in globalBuffer and h.logCh